src/archivist/index.test.js

```diff
@@ -27,14 +26,12 @@ const VERSIONS_PATH = path.resolve(ROOT_PATH, config.get('recorder.versions.stor
 const MIME_TYPE = 'text/html';
 const FETCH_DATE = new Date('2000-01-02T12:00:00.000Z');
 
-let snapshotsStorageAdapter;
-let versionsStorageAdapter;
-
 async function resetGitRepositories() {
-  return Promise.all([ snapshotsStorageAdapter._removeAllRecords(), versionsStorageAdapter._removeAllRecords() ]);
+  return Promise.all([ app.recorder.snapshotsRepository.removeAll(), app.recorder.versionsRepository.removeAll() ]);
 }
 
 let gitVersion;
+let app;
```

Comment:
There is a forward usage of `app` that has not been declared yet. Move these lines right after the `const` declarations.